Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -883,6 +883,8 @@
         - ShellTranquilizer
         - SpeedLoaderMagnum
         - SpeedLoaderMagnumEmpty
+        - MagazineSVT #frontier by VividPups
+        - MagazineSVTempty #frontier VividPups
```

```suggestion
        - MagazineSVT #Frontier
        - MagazineSVTEmpty #Frontier
```

Comments, capitalize empty.

---

Magazine is cap because it's the First letter.  SVT is cap because it's an acronym